docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

```diff
@@ -24,6 +24,85 @@ If Users use the same email address in Okta that they use with other authenticat
+   - **Grant type**: Select **Authorization Code** and **Refresh Token**.
+   - **Sign-in redirect URIs**: Replace the default setting with the Grafana Cloud Okta path, replacing <YOUR_ORG> with the name of your Grafana organization: https://<YOUR_ORG>.grafana.net/login/okta. For on-premises installation, use the Grafana server URL: http://<my_grafana_server_name_or_ip>:<grafana_server_port>/login/okta.
+   - **Sign-out redirect URIs (optional)**: Replace the default setting with the Grafana Cloud Okta path, replacing <YOUR_ORG> with the name of your Grafana organization: https://<YOUR_ORG>.grafana.net/login/okta. For on-premises installation, use the Grafana server URL: http://<my_grafana_server_name_or_ip>:<grafana_server_port>/login/okta.
+   - **Base URIs (optional)**: Add any base URIs
+   - **Controlled access**: Select whether to assign the app integration to everyone in your organization, or only selected groups. You can assign this option after you create the app.
+
+1. Make a note of the following:
+   - **ClientID**
+   - **Client Secret**
+   - **Auth URL**
+     For example: https://<TENANT_ID>.okta.com/oauth2/v1/authorize
+   - **Token URL**
+     For example: https://<TENANT_ID>.okta.com/oauth2/v1/token
+   - **API URL**
+     For example: https://<TENANT_ID>.okta.com/oauth2/v1/userinfo
+
+### Configure Okta to Grafana Cloud role mapping
+
+1. In the **Okta Admin Console**, select **Directory > Profile Editor**.
+1. Select the Okta Application Profile you created previously (the default name for this is `<App name> User`).
+1. Select **Add Attribute** and fill in the following fields:
+
+   - **Data Type**: string
+   - **Display Name**: Meaningful name. For example, `Grafana Role`.
+   - **Variable Name**: Meaningful name. For example, `grafana_role`.
+   - **Description (optional)**: A description of the role.
+   - **Enum**: Select **Define enumerated list of values** and add the following:
+     - Display Name: Admin Value: Admin
+     - Display Name: Editor Value: Editor
+     - Display Name: Viewer Value: Viewer
+
+   The remaining attributes are optional and can be set as needed.
+
+1. Click **Save**.
+1. (Optional) You can add the role attribute to the default User profile. To do this, please follow the steps in the [Optional: Add the role attribute to the User (default) Okta profile]({{< relref "#optional-add-the-role-attribute-to-the-user-default-okta-profile" >}}) section.
+1. In the **Okta Admin Console**, select **Security > API**.
+1. From the **Authorization Servers** tab, select server you want to configure. By default, this is `default`.
+1. On the **Claims** tab, click **Add Claim**.
+1. Configure the claim as follows:
+
+   - **Name**: `grafana_role`
+   - **Include in token type**: Select **ID Token** and **Always**
+   - **Value type**: `Expression`
+   - **Value**: `userapp.grafana_role`
+   - **Included in**: `Any scope`
+
+1. Click **Create**.
+
+### Configure Groups claim
```

Comment:
The [OSS documentation](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/okta/#configure-role-mapping) and the JMESPath expressions there for `role_attribute_path` imply the usage of Okta groups for role assignment instead of a defined `grafana_role`. If this is going to a pivot away from utilizing that method, then the OSS docs will also need to follow suit.